net/isc-dhcp/files/dhcpd.init

```diff
@@ -309,6 +310,32 @@ boot() {
 
 service_triggers()
 {
-	procd_add_reload_trigger "dhcp"
-	procd_add_raw_trigger "interface.*" 3000 /etc/init.d/dhcpd reload
+	if [ -n "$DHCPD_BOOT" ] ; then
+		# Make the first start robust to slow interfaces; wait a while
+		procd_add_raw_trigger "interface.*.up" 5000 /etc/init.d/unbound restart
+
+	else
+		# reload with normal parameters
+		procd_add_reload_trigger "network" "dhcp"
+		config_load dhcp
+		config_foreach add_interface_trigger dhcp
+	fi
 }
+
+reload_service() {
+	rc_procd start_service "$@"
+	prodcd_send_signal dhcpd "$@"
+}
+
+add_interface_trigger() {
```

Comment:
`add_interface_trigger()` probably should be defined above `service_triggers()`

Reply:
Fixed both.